app/services/additional_claimants_csv/validator.rb

```diff
@@ -1,3 +1,6 @@
+class CsvDobValidationError < StandardError
```

Comment:
That would limit that validation error to AdditionalClaimantsCsv class, which I want to avoid.